mappings/net/minecraft/entity/projectile/EntityArrowBase.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/aan net/minecraft/entity/projectile/EntityProjectile
+CLASS none/aan net/minecraft/entity/projectile/EntityArrowBase
```

Comment:
There's projectile classes (fireballs, snowballs, enderpearls, ...) that don't extend this class. `EntityArrowBase` is the base class for normal and specter arrows.